### PR TITLE
Resolves #48 Add One Key Output

### LINKED ISSUE #48: The ability to request only 1 code
have the ability to just get one code, for an example:
```
C:\>gauth -snapchat
snapchat "code" "code" "code"
[ ======                      ]
C:\>
```

### PR DESCRIPTION
**Pull request by thepyzhov**
- Add ability to request only one code for storing it in a clipboard: `gauth Google -b | pbcopy`
- Add new example with parameter and flag to README

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+	"github.com/creachadair/otp/otpauth"
+	"github.com/pcarrier/gauth/gauth"
+	"github.com/spf13/cobra"
+	"golang.org/x/term"
+)
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "gauth",
+	Short: "CLI-Application is for getting OTP codes",
```
Comment (pcarrier) [inline cmd/root.go:22]: "OTP codes from the command line"?

Comment (thepyzhov) [reply cmd/root.go]: Done

**Comment by pcarrier** [comment]
This doubles the binary size. Could you do without the seemingly large lib?

**Comment by thepyzhov** [comment]
> This doubles the binary size (from ~2MB to ~4MB). Could you do without the seemingly large lib?

@pcarrier done. Removec Cobra

**Review by pcarrier — APPROVED** [review]

**Pull request merged**